exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/CryptocurrencyService.java

```diff
@@ -26,4 +29,10 @@
   Optional<Wallet> getWallet(PublicKey ownerKey);
 
   List<HistoryEntity> getWalletHistory(PublicKey ownerKey);
+
+  void createWalletTx(TxMessageProtos.CreateWalletTx arguments, TransactionContext context)
```

Comment:
(Here and in similar cases) 
I'd drop "Tx" suffix (just createWallet — a service operation).